yabt/builders/cpp.py

```diff
@@ -253,9 +253,13 @@ def compile_cc(build_context, compiler_config, buildenv, sources,
         obj_rel_path = '{}.o'.format(splitext(src)[0])
         obj_file = join(buildenv_workspace, obj_rel_path)
         include_paths = [buildenv_workspace] + compiler_config.include_path
+
+        # ${buildenv_workspace} is replaced by the actual value at
+        # compilation time
```

Comment:
Please add some more documentation (with the use case at hand).
It took me time to understand that ```${...}``` is a language this PR introduce for "late evaluation".
I think I would have used a different less general looking (the reader can expect other internal variable to work too).
(e.g. ```__BUILDENV_WORKSPACE__```)

Either way communicates with the user of ybt with a very internal, implementation detail dependant variable.

I think I would like better a direct support for -fdebug-prefix-map flag - which is much simpler
we can introduce a use_debug_prefix_map control variable in get_common_config so ybt can support compilers without this feature

Reply:
I added a new flag to YBT.   I update the comment to this PR (above), added a new test in cpp_test.py and a long comment that explains the flag.